src/components/TabBar.js

```diff
@@ -151,12 +158,12 @@ export default function BasicTabs({ modalData, height, weight, abilities, moveSe
 
         {/****** EVOLUTION TAB ******/}
```

Comment:
I like the comments to separate different components

Reply:
me too! this one took a lot of time to plan out